src/components/VItemGroup/VItem.vue

```diff
@@ -18,7 +18,7 @@
     <VButton
       data-item-group-item
       :as="as"
-      class="group relative flex min-w-full justify-between py-2 hover:bg-dark-charcoal-10 focus:z-10"
+      class="group relative flex min-w-full justify-between border-0 py-2 hover:bg-dark-charcoal-10 focus:z-10"
```

Comment:
Removing the border here makes this a special variant of Button that's only defined here, along with its usage. This might cause issues later when we try to update the button component based on @panchovm's plans.